torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,46 @@
+import io
+from typing import Optional
+
+import torch
+
+from .export_options import ExportOptions
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
+    /,
+    *args,
+    options: Optional[ExportOptions] = None,
+    **kwargs,
+) -> ExportOutput:
+    """Export a torch.nn.Module to a ONNX graph
+
+    Args:
+        torch_model: User model to be exported
+        args: Positional inputs to `torch_model`
+        options: Advanced export options
```

Comment:
I'm not sure we want to imply these are advanced... they're just options. Users should feel free to explore the class to influence their export.

Reply:
lgtm